consensus/src/counters.rs

```diff
@@ -507,13 +507,19 @@ pub static NUM_BLOCKS_IN_PIPELINE: Lazy<IntGaugeVec> = Lazy::new(|| {
 //     .unwrap()
 // });
 
+const NUM_CONSENSUS_TRANSACTIONS_BUCKETS: [f64; 20] = [
+    5.0, 10.0, 20.0, 40.0, 75.0, 100.0, 125.0, 150.0, 175.0, 200.0, 250.0, 300.0, 350.0, 400.0,
```

Comment:
Do we need 250, 350, 450 buckets? Also, with Quorum Store, we might have up to 20k transactions per block, so we need to accommodate for that here. Its very possible we will hit that number often in forge at least.